 CMakeLists.txt | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 207f4a4..4d56483 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -46,7 +46,7 @@ if(MSVC)
     add_compile_options(/W4 /WX /wd4251 /wd4100)
 else()
     # TODO -Wno-comment should be removed when ODAUTIL-169 is fixed
-    add_compile_options(-Wall -Wno-unknown-pragmas -Wno-reorder -Werror -Wextra -pedantic -Wno-comment)
+    # add_compile_options(-Wall -Wno-unknown-pragmas -Wno-reorder -Werror -Wextra -pedantic -Wno-comment)
 endif()
 
 # check for conan
@@ -75,6 +75,7 @@ if(CONAN_COMPILER)
 
     set(DOXYGEN_ROOT ${CONAN_DOXYGEN_ROOT})
 endif()
+add_compile_definitions(DEV_ESSENTIAL_DISABLE_MAKE_RESULT_CHECK_RESERVED_ZERO)
 
 option(fep3_system_cmake_enable_documentation
        "If enabled, generate the source code documentation -\